### ISSUE TITLE
Timestamp of comment uses user's system time settings.

### ISSUE THREAD

**Issue by aaronang** [devhub-tud/devhub#229]
As you can see in the picture below, the timestamp of a comment uses the user's system time setting. I think it makes more sense to display the server time.
![time-difference](https://cloud.githubusercontent.com/assets/7579804/8220741/bf5f12c2-1556-11e5-8fcd-21f3b25da09a.png)

**Comment by JWGmeligMeyling** [comment]
Is this right after commenting (javascript) or after a page refresh?

**Comment by aaronang** [comment]
I see, after a page refresh it is correct :smile:

**Comment by JWGmeligMeyling** [comment]
I want to fix this some day but its not worth the effort at the moment.
Comments are currently rendered by Freemarker and the date is formatted for the browsers preferred Locale. This is however difficult to integrate with the REST api used for posting the comments, therefore we use the JS current time as a quick fix.VLater on we will probably leave the rendering of the comments to AngularJS or Web components, which will also take over the date format.

**Comment by aaronang** [comment]
I guess I will leave this issue open?

**Comment by JWGmeligMeyling** [comment]
Yes that is OK, the inconsistency between the javascript injected comments and freemarker comments are a known issue though ;) And I might fix this when implementing Freemarker but I first want to find out a better way to manage web dependencies.

Btw,  I believe you can XSS some fancy colours into your comments by abusing this very same problem. You won't be able to harm others though ;)

**Comment by aaronang** [comment]
Not sure, but in class nl.tudelft.ewi.devhub.server.web.resources.ProjectResource, I see a line which returns a CommentResponse with the date set to server time? This CommentResponse is returned to the AJAX call [here](https://github.com/devhub-tud/devhub/blob/8354bf0572a56e0f6e32d8eed035f5b7d13dcf38/src/main/resources/templates/components/inline-comments.ftl#L43). Afterwards you are creating the comment container [here](https://github.com/devhub-tud/devhub/blob/8354bf0572a56e0f6e32d8eed035f5b7d13dcf38/src/main/resources/templates/components/inline-comments.ftl#L60). This should display the server time right (res.date)? Or am I missing something? Where is the part where it sets the current time?

**Comment by JWGmeligMeyling** [comment]
The `ProjectResource` is responsible for the comment backend endpoint: https://github.com/devhub-tud/devhub/blob/master/src/main/java/nl/tudelft/ewi/devhub/server/web/resources/ProjectResource.java#L223 . It uses the toString of the date apparently.

**Comment by JWGmeligMeyling** [comment]
I am also confused by this haha. Maybe there is a difference in implementation between commit/pr comments and inline comments
